console-subscriber/src/lib.rs

```diff
@@ -226,6 +243,28 @@ where
         }
     }
 
+    fn on_event(&self, event: &tracing::Event<'_>, _ctx: Context<'_, S>) {
+        let meta = event.metadata();
+        // make faster like spawn metadata pointer check?
```

Comment:
hmm, i think if we wanted to do that, we would need to store a separate metadata pointer for _each_ waker event. also, that would rely on the assumption that there's only one callsite for each waker op?

Reply:
Yea exactly. One thing that I noticed _is_ staying the same is the `meta.target()` pointer, since that static string is only in 1 place in the Tokio crate. But it could also be fragile...